zone/client.h

```diff
@@ -725,6 +733,23 @@ class Client : public Mob
 	void SendGuildList();
 	void SendGuildJoin(GuildJoin_Struct* gj);
 	void RefreshGuildInfo();
+	void SendGuildRankNames();
+	void SendGuildTributeDetails(uint32 tribute_id, uint32 tier);
+	void DoGuildTributeUpdate();
+	void SendGuildActiveTributes(uint32 guild_id);
+	void SendGuildFavorAndTimer(uint32 guild_id);
+	void SendGuildTributeOptInToggle(const GuildTributeMemberToggle* in);
+	void RequestGuildActiveTributes(uint32 guild_id);
+	void RequestGuildFavorAndTimer(uint32 guild_id);
+	void SendGuildMembersList();
+	void SendGuildMemberAdd(uint32 guild_id, uint32 level, uint32 _class, uint32 rank, uint32 spirit, uint32 zone_id, std::string player_name);
```

Comment:
![image](https://github.com/EQEmu/Server/assets/3319450/d53df3c4-8fc7-4155-9eaf-0338ecd240cb)

All of these (in gray) are un-used - what's the story here ?

Reply:
![image](https://github.com/EQEmu/Server/assets/3319450/6d8ee883-5c12-49bb-8f03-183983f973b5)

Reply:
These are the result of adding single entry opcodes for client though I did not use all of them as several are entity based.  They are present in case they are required.  I would suggest that they will be used as I work through the items noted below, for single opcode changes instead of sending the guild member list each time.